recipe/build.sh

```diff
@@ -63,7 +63,8 @@ rm -rf $PREFIX/bin/protoc
 # Then find the .gn or .gni files that these ninja files were created from and figure out wtf is going on.
 
 # qtwebengine needs python 2
-conda create -y --prefix "${SRC_DIR}/python2_hack" -c https://repo.continuum.io/pkgs/main --no-deps python=2
+# conda create -y --prefix "${SRC_DIR}/python2_hack" -c https://repo.continuum.io/pkgs/main --no-deps python=2
```

Comment:
Its probably not worth rebuilding with this line removed, but maybe we can remember to remove it in the future.

Reply:
It's already commented out, isn't it?